modules/microshift-storage-vol-snapshot-restore.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// microshift/creating-volume-snapshots-microshift.adoc
+
+:_content-type: PROCEDURE
+[id="restoring-vol-snapshot-microshift_{context}"]
+= Restoring a volume snapshot
+
+The following workflow demonstrates snapshot restoration. In this example, the verification steps are also given to verify that data written to a source PVC is preserved and restored on the new PVC.
+
```

Comment:
Can we add a note somewhere is this section saying, "when restoring a PVC with snapshot datasource, the requested PVC storage capacity should be equal to that of source volumeSnapshot, else restoring a volume snapshot fails".

Reply:
@radeore Can the destination storage be larger than the source, or does it have to be truly, exactly equal?

Reply:
Yeah, that's correct @radeore